src/router.js

```diff
@@ -220,4 +223,11 @@ export default new Router({
       ]
     }
   ]
-})
\ No newline at end of file
+});
+
+router.beforeEach((to, from, next) => {
```

Comment:
My fault, this method already defined in main.js, could you refactor this code?

Reply:
fixed ❤️